sql/migrations/20180416111715_create_bots_table.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('bots', function (table) {
+    table.increments()
+    table.string('name').unique()

```

Comment:
hm, whether the name should be unique is a good question. i think the name should be anything but pubs should have a "slug" (what will be in the url, like ${slug}.butt.nz) and a domain ("butt.nz" is the only domain to start, i have a few others ready), then the pubs should be unique by slug and domain.

http://knexjs.org/#Schema-unique